code/modules/spells/spell_types/hivemind.dm

```diff
@@ -389,7 +389,7 @@
 			backseat.ghostize(0)
 		else
 			backseat.mind.transfer_to(vessel,1)
-	vessel.visible_message("<span class='userdanger'>[src] suddenly wakes up, as though he was under foreign control!</span>")
+	vessel.visible_message("<span class='userdanger'>[vessel] suddenly wakes up, as though he was under foreign control!</span>")
```

Comment:
I'm no english major, so I don't know if using the past tense is actually correct in the latter part of this message, but it certainly isn't correct to always use the male genus.
```
	vessel.visible_message("<span class='userdanger'>[vessel] suddenly wakes up, as though [vessel.p_they()] [vessel.p_were()] under foreign control!</span>")